src/sqlfluff/dialects/dialect_postgres_keywords.py

```diff
@@ -203,7 +203,7 @@ def get_keywords(keyword_list, keyword_type):
     ("CURRENT_TIME", "reserved"),
     ("CURRENT_TIMESTAMP", "reserved"),
     ("CURRENT_TRANSFORM_GROUP_FOR_TYPE", "not-keyword"),
-    ("CURRENT_USER", "reserved"),
+    ("CURRENT_USER", "non-reserved"),
```

Comment:
`USING (ename=current_user)` is common in CREATE POLICY statements.
I needed to set this to non-reserved in order to get the expression above to parse. Let me know if this is incorrect/there's a better method 😄

Reply:
If you want to overwrite behaviour that is specified in the docs - please can you add it to the postgres_nondocs_keywords lower down. They take precedence, and I like to keep parity with the docs wherever possible

Reply:
@WittierDinosaur thanks, moved the keyword down to `postgres_nondocs_keywords` now so this should be good to go 🚀